scripts/create-local-tag-for-release

```diff
@@ -1,16 +1,19 @@
 #!/bin/bash
 
 # Script to create a new local tag in preparation for a release
+# This script is idempotent i.e. it always fetches remote tags to create an accuarate new tag.
```

Comment:
spelling: accuarate --> 'accurate'  (same as below)

nit: should the i.e. be more something like "always fetches and creates the next eligible|logical (not sure which word) tag"? idempotent means no matter how many times you run it you should get the same, valid tag right?

Reply:
made it "it always fetches remote tags to create the new tag". 

re: idempotency -- yes, for e.g If the current remote release tag is v1.0.0, 
running `create-local-tag-for-release -p` will create a new tag v1.0.1, 
immediately running `create-local-tag-for-release -m` will create a new tag v2.0.0, 
subsequently rerunning `create-local-tag-for-release -p` will create a new tag v1.0.1